metadata/Passports__16__metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "changeLog":  "",
+    "cpu":  1,
+    "description":  "Machine Learning model for extracting commonly occurring data points from Passports, including header fields and line items. Please see more details including supported languages in the ML Packages section of the Document Understanding Guide: https://docs.uipath.com/document-understanding/docs/ml-packages",
+    "displayName":  "Passports",
+    "gpu":  0,
+    "inputDescription":  "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -\u003e Other Services view.",
+    "inputType":  "JSON",
+    "memory":  0,
+    "mlPackageLanguage":  "PYTHON37_DU",
+    "name":  "Passports",
+    "outputDescription":  "Please refer to the documentation of the Activity used to query the ML Skill.",
+    "processorType":  "GPU",
```

Comment:
this is the only head / doctype that says `"processorType": "GPU"` (all the rest say `"CPU"`), is this ok? 👀

Reply:
It is informative only. Fixed!